tests/python/23_bandwidth/host_mem_23_bandwidth.py

```diff
@@ -0,0 +1,211 @@
+"""
+ Copyright (C) 2018-2020 Xilinx, Inc
+
+ Python OpenCL based bandwidth testcase used with every platform as part of
```

Comment:
we have removed the support for pyopencl. please refer to tests/python/23_bandwidth/versal_23_bandwidth.py for your example.